### PR TITLE
109 Add README.md

### PR DESCRIPTION
**Pull request by GeorgiChochov**
Just an initial README, barebones. Focus should be on making it clear how to build and run. We will provide more extensive documentation whit first release in Project Pages and Wiki.

### CODE REVIEW

**Comment by GeorgiChochov** [comment]
@hamc17, you had the most recent experience building and deploying, please let me know if I've missed anything.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+`mvn -DskipTests clean package`
+
+## <a name="run"/>How to run
+#### Menas requirements:
+- **Tomcat 8.5/9.0** installation
+- **MongoDB 4.0** installation
+- **HADOOP_CONF_DIR** environment variable, pointing to the location of your hadoop configuration
+
+#### Deploying Menas
+Simply copy the **menas.war** file produced whn building the project into Tomcat's webapps directory.
```
Comment (Zejnilovic) [inline README.md]: whn -> when

File: README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+--class za.co.absa.enceladus.standardization.StandardizationJob \
+<standardization-<build-version>.jar> \
+--dataset-name <Dataset Name> \
+--dataset-version <Dataset Version> \
+--report-date <Date> \
+--report-version <Data Run Version> \
+--raw-format <data format> \
+--row-tag <tag> \
+--header <true/false(applicable for CSV)> \
+--delimiter <csv delimiter (applicable for CSV)>
```
Comment (Zejnilovic) [inline README.md]: Having both row-tag and header + delimiter here can be misleading. I propose to add a standalone section about file formats and options there.

File: README.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ Simply copy the **menas.war** file produced whn building the project into Tomcat
 #### Standardization and Conformance requirements:
 - [**Spark 2.2.1-ABSA**](https://github.com/AbsaOSS/spark/tree/branch-2.2.1-ABSA) installation (this custom version can be [built like any other Spark version](https://spark.apache.org/docs/latest/building-spark.html))
 - **Hadoop 2.7** installation
+- **Menas Credentials File** (a configuratino file containing a username and password for the Spark jobs to authenticate with Menas)
```
Comment (Zejnilovic) [inline README.md]: configuratino -> configuration

File: README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+<conformance_<build_version>.jar> \
+--menas-credentials-file <path_to_menas_credentials>
+--dataset-name <dataset_name> \
+--dataset-version <dataset_ersion> \
+--report-date <date> \
+--report-version <data_run-version>
+```
+
+## <a name="contribute"/>How to contribute
+Please see our [**Contribution Guidelines**](https://github.com/AbsaOSS/enceladus/blob/develop/CONTRIBUTING.md).
```
Comment (benedeki) [inline README.md]: I think a relative link would be better

File: README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+**Enceladus** is a **Dynamic Conformance Engine** which allows data from different formats to be standardized to parquet and conformed to group-accepted common reference (e.g. **DE** in one source system and **Deutschland** in another, can be conformed to **Germany**).
+
+The project is comprised of three main components:
+### <a name="menas"/>Menas
+This is the user-facing web client, used to **specify the standardization schema**, and **define the step required to conform** a dataset.  
+
+### <a name="standardization"/>Standardization
+This is a Spark job which reads in a dataset in any supported format and **produces a parquet dataset with the Menas-specified schema**. 
+
+### <a name="conformance"/>Conformance
```
[the remark this answers is not in the document]
Comment (GeorgiChochov) [reply README.md]: I don't want to explain this too much right now, this is just an initial README. We'll provide proper documentation with first release. Right now the focus is on making sure we have a general outline of how to build and run.

[the remark this answers is not in the document]
Comment (benedeki) [reply README.md]: OK 👍

**Pull request merged**